.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Review App Build
```

Comment:
Shall we call this workflow file `review-mobile-app.yml` or something?

Reply:
Addressed in 34b8308